### ISSUE TITLE
Wheel repair doesn't work correctly on RHS HMMWV

### ISSUE THREAD

**Issue by MAD-BUG** [acemod/ACE3#2742]
Repairing of HMMWV wheels work like they are standart parts of car. You can only repair them with toolkit without consumpting spare wheels.

**Comment by jokoho48** [comment]
this is a RHS Fault that we cant fix

**Comment by MAD-BUG** [comment]
can you say in which place there is a fault?

**Comment by PabstMirror** [comment]
RHS configs are very odd but I can't say for certain that they are wrong.

This wheel:
```
    class LR: LF {
        boneName = "wheel_1_2_damper";
        center = "axis_wheel_3_1";
    };
```

matches with this hitpoint:
```
class HitLF2Wheel {
    name = "wheel_3";
    visual = "wheel_3_tire";
};
```
Looks like they mix `wheel_X_Y` with `wheel_N`.

I added a backup section to handle this and search for the closest hitpoint to the wheel's center.

**Comment by sargken** [comment]
Anyone Report this to RHS?

**Comment by MAD-BUG** [comment]
This was writen before i get confirmation about this here. http://feedback.rhsmods.org/view.php?id=1280#c2578

**Comment by sargken** [comment]
So what is the problem we are facing. What needs to be changed? on RHS side.

**Comment by PabstMirror** [comment]
Fixed via #2861 

The way everyother car does it is to have the hitpoint selection be a varient of the wheel's boneName. So boneName `wheel_1_1_damper` can be matched with selection `wheel_1_1_steering`.

RHS's HMMVWES have wheel selections `["wheel_1", "wheel_2", "wheel_3", "wheel_4"]`, which isn't "wrong", but is non-standard.

```
//Normal Car:
_wheelName=LF, _wheelCenter=wheel_1_1_axis, _wheelBone=wheel_1_1_damper, _wheelBoneNameResized=wheel_1_1
//will match:
_wheelHitPoint=HitLFWheel,  _wheelHitPointSelection=wheel_1_1_steering

//RHS HMMV:
_wheelName=LF, _wheelCenter=axis_wheel_1_1, _wheelBone=wheel_1_1_damper, _wheelBoneNameResized=wheel_1_1
//will match:
_wheelHitPoint=HitLFWheel,  _wheelHitPointSelection=wheel_1
```
